test/embedded-refract-test.js

```diff
@@ -0,0 +1,442 @@
+var _ = require('lodash');
+var expect = require('./spec-helper').expect;
+var minim = require('../lib/minim');
+
+describe('Embedded Refract', function() {
+  context('when converting from embedded refract', function() {
+    context('when given a normal object', function() {
+      var obj = {
+        foo: 'bar'
+      };
+
+      var element;
+
+      before(function() {
+        element = minim.convertFromEmbedded(obj);
+      });
+
+      it('returns the correct value', function() {
+        expect(element.toValue()).to.deep.equal(obj);
+      });
+    });
+
+    context('when given embedded refract object', function() {
+      // An object { b: 2 } with embedded refract
+      var obj = {
+        refract: {
```

Comment:
Hm. This seems a bit strange. So the idea is that if I take a non-refracted object, I can put a refract element in it that provides metadata about the object itself? Seems a bit of a mind-tweak.